src/main/java/gtclassic/recipe/GTRecipeUUAmplifier.java

```diff
@@ -44,22 +45,18 @@ public static void init() {
 	 * Stuff below for actual amp recipes and JEI iterators
 	 **/
 	public static IRecipeModifier[] value(int amount) {
-		return new IRecipeModifier[] { ModifierType.RECIPE_LENGTH.create((amount / 1) - 100) };
+		return new IRecipeModifier[] { ModifierType.RECIPE_LENGTH.create(amount - 100) };
 	}
 
 	public static void addAmplifierToJei(IRecipeInput[] inputs, IRecipeModifier[] modifiers, ItemStack... outputs) {
 		List<IRecipeInput> inlist = new ArrayList<>();
 		List<ItemStack> outlist = new ArrayList<>();
```

Comment:
Can be reduced to ObjectArrayList.wrap(array) so way less code.

Reply:
fixed in ec4dd3fc6fe61fa075c8f98b06a70e0b3471871c and 4b65ed77f80277727fce24f4501157398784c45a